backend/experiment/templates/collection-dashboard.html

```diff
@@ -0,0 +1,310 @@
+{% extends "admin/base_site.html" %} {% block content %}{% load static %}
+<script src="{% static 'collection_dashboard_sessions.js' %}"></script>
+<style>
+    input {
+        margin-right: 1rem;
+    }
+    input[value="close"] {
+        float: right;
+
+    }
+    table {
+        width: 100%;
+        margin-bottom: 3rem;
+    }
+    #all-sessions tr {
+        background-color: var(--body-bg)
+    }
+    td, th {
+        vertical-align: middle;        
+        width: auto;        
+    }
+    td {
+        padding-left: 1rem;
+        padding-right: 1rem;
+    }
+    td.table-sessions {
+        width: 6rem;
+    }
+    td.table-results {
+        width: 8rem;
+    }
+    td.participants-overview {
+        max-width: 25vw;        
+    }
+    pre {
+        color: black;
+        font-size: 1rem;
+    }
+    .hide {
+        display: none;
+    }
+    .dashboard-caption {
+        font-weight: 600;
+        margin: 2rem 0px 2rem 0px;
+    }
+    .pop-up {        
+        background-color: var(--body-bg);
+        box-shadow: 0px 0px 10px var(--primary);
+        position: fixed;
+       
+        padding: 2rem;        
+        border-style: solid;
+        border-color: var(--primary);
+        border-width: 1px;   
+        border-radius: 1px;
+        overflow-y: scroll;
+    }
+    .pop-up-big {
+        top: 4.2rem;
+        left: 10vw;        
+        width: calc(80vw - 34px);
+        height: 80vh;        
+
+    }
+    .pop-up-small {
+        top: 6.2rem;
+        left: 12vw;
+        width: calc(76vw - 51px);
+        height: 70vh;
+    }
+</style>
+
+<h1>Experiments in collection: {{ collection.name }}</h1>
+<p>
+    <form action="" method="post">
+        <!-- Show all participants and sessions -->        
+        <input type="button" value="{{ collect_data.participant_count }} participants" id="show-participants" />
+        <input type="button" value="{{ collect_data.session_count }} sessions" id="show-sessions" />
+    </form>   
+</p>
+
+<!-- Experiments table -->
```

Comment:
The separate tables might be in their own html file, and included here. That way, tweaking a specific table is going to be easier.

```
{% include "subtemplate.html" %}
```

Reply:
Good point!